Tools/cases_generator/generate_cases.py

```diff
@@ -1249,12 +1249,11 @@ def main():
     args = arg_parser.parse_args()  # Prints message and sys.exit(2) on error
     if len(args.input) == 0:
         args.input.append(DEFAULT_INPUT)
-    if args.cheri_bits:
-        # Ensure that this is in sync with Includes/internal/pycore_code.h and Lib/opcode.py
-        # Use 2 * pointer width because of alignment, the size is in code units
-        cache_effect_ptr_size = int(2 * args.cheri_bits / BITS_PER_CODE_UNIT)
-    else:
-        cache_effect_ptr_size = 4
+
+    # Ensure that this is in sync with Includes/internal/pycore_code.h and Lib/opcode.py
+    # Use 2 * pointer width because of alignment, the size is in code units
+    cache_effect_ptr_size = 16 # 2 * 8
```

Comment:
I think the `2 * 8` comment is redundant here. Perhaps expand the comment above to mention that the cache_effect_ptr_size is in "code units", which are 2 bytes wide, so this translates to 2 * max_architectural_pointer_width.